### ISSUE TITLE
ListView has a one pixel line between elements

### ISSUE THREAD

**Issue by Vardiak** [flutter/flutter#37168]
## Steps to Reproduce

When using a ListView where each elements have it's own background color, the list sometime has a one pixel transparent line between elements.

```
class TestScroll extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Container(
      color: Colors.white,
      child: ListView(
        children: <Widget>[
          for (var i = 0; i < 50; i++)
            Container(
              height: 50,
              color: Colors.black,
            )
        ],
      ),
    );
  }
}
```

![Screenshot_20190729-131100](https://user-images.githubusercontent.com/19309601/62044385-1c3e4600-b203-11e9-88c3-dbfc3e043b6f.png)

## Logs

There is no meaning full log for this issue.

```
[✓] Flutter (Channel beta, v1.7.8+hotfix.4, on Mac OS X 10.14.6 18G84, locale en-FR)
    • Flutter version 1.7.8+hotfix.4 at /Users/***/Documents/flutter
    • Framework revision 20e59316b8 (10 days ago), 2019-07-18 20:04:33 -0700
    • Engine revision fee001c93f
    • Dart version 2.4.0


[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.1)
    • Android SDK at /Users/***/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.1
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 10.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.3, Build version 10G8
    • CocoaPods version 1.7.5

[✓] iOS tools - develop for iOS devices
    • ios-deploy 1.9.4

[✓] Android Studio (version 3.4)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 37.1.1
    • Dart plugin version 183.6270
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)

[✓] VS Code (version 1.36.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.2.0

[✓] Connected device (1 available)
    • Nokia 7 plus • 192.168.0.216:5555 • android-arm64 • Android 9 (API 28)

• No issues found!
```

**Comment by Vardiak** [comment]
This problem seems to be limited to Android devices since I couldn't reproduce it using the iPhone Simulator

**Comment by HansMuller** [comment]
This looks like a known problem with antialiasing and mapping from logical pixels to device pixels. There's an open issue for a documentation fix: #14288

More info about the problem can be found here:
#14288 (comment)
#17084 (comment)
#15035 (comment)

**Comment by escamoteur** [comment]
I close this because of the other issues
